contributingGuides/STYLE.md

```diff
@@ -112,9 +672,9 @@ useEffect(() => {
 
 ```
 
-Empty functions (noop) should be declare as arrow functions with no whitespace inside. Avoid _.noop()
+Empty functions (noop) should be declared as arrow functions with no whitespace inside. Avoid _.noop()
```

Comment:
This rule is useless imo, handled automatically with eslint + we no longer use lodash and underscore so we don't have to worry about _.noop usage